_getting-started/сontracts-and-workspaces.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Contracts and Workspaces
+layout: article
+section: Basic Concepts
+order: 1
+since: 20190328
+---
+
+This document provides information on [contracts](#contracts), [contract
+management](#contract-management), [workspaces](#workspaces) and [workspace
+management](#section). Additionally, it explains the basic application of this
+approach in [solution life cycle](#workspaces-and-life-cycles). The following
+scheme shows here contracts and workspaces stand in solution hierarchy.  
+
+
+
+![](media/73655d79cfa16743bcf8248345ea618b.png)
+
+## Contracts
+
+
+A client’s enclosed environment within a
+[tenant](https://docs.elastic.io/getting-started/tenant.html) is called a
+contract. It is usually backed by a formal agreement, hence the name. Each
+client can have multiple contracts. A contract includes members, developer
+teams, and workspaces:
+
+-   A **workspace** is a smaller enclosed environment that contains integration
+    flows. Details on workspaces can be found [here](#workspaces).
+
+-   A **member** is a registered platform user that has been invited the
+    contract by Owner and given a contract
+    [role](https://docs.elastic.io/guides/managing-user-roles-in-a-tenant.html).
+
+-   A **developer team** is a smaller enclosed environment that contains
```

Comment:
Not GitHub, it's just Git. But maybe we should just talk about components repositories.